docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -63,6 +56,20 @@ $ sudo usermod -a -G libvirtd <username>
 $ newgrp libvirtd
 ----
 
+. As root, install the KVM driver binary and make it executable as follows (For Ubuntu-16.04 and higher):
++
+----
+# curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-ubuntu16.04 -o /usr/local/bin/docker-machine-driver-kvm
+# chmod +x /usr/local/bin/docker-machine-driver-kvm
+----
+
+. For Ubuntu-14.04:
```

Comment:
It should be s/Ubuntu-14.04/Ubuntu/. Then add a note that only Ubuntu version 14.04 or higher is supported.

Reply:
different instructions for different version. Here 14.04, and previously 16.04 or higher

Reply:
@gbraad I am resolving this comment and will add new one.